### ISSUE TITLE
Calendar (Needs Management)

### ISSUE THREAD

**Issue by road2nowhere** [where2help/creative#3]
Hi @metakermit and @klappradla this is a quick sketch of how we could improve calendar usability without any pop-ups. Basically with the right Wording (working on it, haha) and some extra lines of text (which might be hidden again).  Just to show you the idea.

![w2h_calendar](https://cloud.githubusercontent.com/assets/15609804/10935715/288fb59a-82e8-11e5-9eb7-83a321da16d9.png)

**Comment by klappradla** [comment]
What should `Agenda` stand for?

Plus: why should we integrate the additional `+ Neues Gesuch erstellen` button? For user's who don't want to go with the drag-drop interface? I could actually see potential for this, the iCal has it too for instance.
![screen shot 2015-11-05 at 01 30 31](https://github-cloud.s3.amazonaws.com/assets%2F3491815%2F10956572%2Fe610aca4-835c-11e5-8f0f-3bd78d5c7940.png)

**Comment by road2nowhere** [comment]
Regarding AGENDA:
It's the same as in Google Calendar (screenshot below): Puts everything in a list which is quiet a convenient view if you have a busy day. I consider it a NICE TO HAVE though, doesn't need to be in the MVP if too much effort.

Regarding the NEUES-GESUCH button, I'd really recommend to have it for 2 reasons:
1. It actually helps to communicate what you can do on this page
2. It makes data entry much easier if you have several overlapping events in a day. 

Regarding DETAIL VIEW:
You mentioned that you are working on the detail view for multiple categories. Could you maybe post a screenshot of the work in progress? 
Looking at the current detail view, I suggest to separate the date entry from the time entry, because it's  not so easy to understand how you can edit the time. (I think it's a bit uncommon that you also have an editable field inside a drop down). I'd suggest to keep it safe and use 2 separate fields for date and time. Maybe you changed that already in the new version?

![image](https://cloud.githubusercontent.com/assets/15609804/10964259/43053012-83a6-11e5-9919-1057d07eb7ca.png)

**Comment by road2nowhere** [comment]
Here are the updated screens.

Calendar view
- general layout as is
- reordering of information in the entry fields
- progress indicator to give at first glance an idea of how many people are still needed (nice to have)

![w2h_calendar_week_101115](https://cloud.githubusercontent.com/assets/15609804/11057830/a81ad2d6-878e-11e5-908c-95ac1246248b.png)

Agenda/List view
- Alternative view of calendar for better readability and tracking of needs (how many people have signed up)

![w2h_calendar_list_101115](https://cloud.githubusercontent.com/assets/15609804/11057705/9179e3c4-878d-11e5-9065-a725e461b00c.png)

Calendar entry detail
- separate fields for specifying a time
- Current request status tracking (nice to have)
- Preview function (nice to have)
- Copy function (nice to have)

![w2h_calendar_detail_101115](https://cloud.githubusercontent.com/assets/15609804/11057760/fa177810-878d-11e5-8150-78a26e7eba88.png)

**Comment by metakermit** [comment]
Looks good and I agree. With the help tooltips, I didn't necessarily mean
they have to be hovering - they could be written as a text paragraph like
you put it. I'd just enable removing them, so that it doesn't take up
screen reel estate every time a user opens this page (cause you generally
only need to figure out how to use it once).

**Comment by metakermit** [comment]
Added something that I found resembles the Agenda view the most from the available [fullcalendar views](http://fullcalendar.io/docs/views/Available_Views/).

![screenshot 2015-12-04 19 50 16](https://cloud.githubusercontent.com/assets/1818628/11598145/76028380-9ac0-11e5-964f-6ead66d98123.png)

**Comment by metakermit** [comment]
Fixed header to look more or less how you described it, @loncie. We added a link to manually create a need without using the drag & drop functionality. I didn't put the arrows on either sides of the date, because the date text varies which makes the buttons jump around (I followed the way they are placed in OS X calendar).

![screenshot 2015-12-04 20 33 04](https://cloud.githubusercontent.com/assets/1818628/11599327/7afb9b78-9ac6-11e5-928b-0577a8dc8d55.png)
